scripts/babel-relay-plugin/src/__tests__/testschema.rfc.json

```diff
@@ -7,6 +7,7 @@
       "mutationType": {
         "name": "Mutation"
       },
+      "subscriptionType": null,
```

Comment:
Should I generate the subscription type here (by adding `Subscription` name to a couple of `utilities.buildASTSchema()` calls next to `Mutation`).